pkg/awx/connection.go

```diff
@@ -260,6 +260,13 @@ func (c *Connection) rawGet(path string, query url.Values) (output []byte, err e
 	if err != nil {
 		return
 	}
+	if 300 <= response.StatusCode {
```

Comment:
Actually I think that we can be more strict here: anything other than 200 is unexpected and we don't currently know how to handle it, so:

```go
if resonse.StatusCode != 200 {
  ...
}
```

In the rest of the code, when the returned parameters have names, we are using just `return`:

```go
err = fmt.Errorf(...)
return
```

Can you do the same here?